Libraries/Image/ImageBackground.js

```diff
@@ -42,11 +43,24 @@ const View = require('View');
  * ```
  */
 class ImageBackground extends React.Component {
+  setNativeProps(props: Object) {
+    if (this._viewRef) {
+      ensureComponentIsNative(this._viewRef);
+      this._viewRef.setNativeProps(props);
+    }
+  }
+
+  _viewRef: View;
+
+  _captureRef = ref => {
```

Comment:
There's some difference with function binding, which can be expensive. Doing it like this will guarantee it's only bound once I believe.